src/universal/routes/welcome.js

```diff
@@ -1,8 +1,28 @@
-export default {
+import makeReducer from 'universal/redux/makeReducer';
+import {resolvePromiseMap} from 'universal/utils/promises';
+
+const setImports = () =>
+  new Map([
+    ['component', System.import(
+      'universal/modules/welcome/containers/Welcome/Welcome')],
+    ['reduxForm', System.import('redux-form')],
+    ['welcome', System.import('universal/modules/welcome/ducks/welcomeDuck')]
+  ]);
+
+const getImports = importMap => ({
+  component: importMap.get('component'),
+  reduxForm: importMap.get('reduxForm'),
+  welcome: importMap.get('welcome').default
```

Comment:
lazy loading reducers is great, but then we can't persist the results (at least not until a new `combineReducers` comes out). Not a huge deal though, since we'll almost never want to persist local-state because persisting something that could potentially be the source of a bug is walking a thin line.